keel-igor/keel-igor.gradle.kts

```diff
@@ -9,6 +9,8 @@ dependencies {
   implementation("org.springframework.boot:spring-boot-autoconfigure")
   implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core")
 
+  implementation ("io.github.resilience4j:resilience4j-kotlin")
```

Comment:
it's not recognizing `io.github.resilience4j.kotlin.retry.decorateSuspendFunction` in `ArtifactMetadataService` without adding this here.